src/data/CONTRIBUTORS.js

```diff
@@ -37,5 +37,11 @@ export default {
 		name: 'Jump',
 		avatar: process.env.PUBLIC_URL + '/avatar/jump.png',
 		jobs: [JOBS.RED_MAGE],
-	},
+		}
+	FURST: {
+		name: 'Fürst',
+		avatar: process.env.PUBLIC_URL + '/avatar/Furst.png',
+		jobs: [
+			JOBS.BLACK_MAGE,]}
```

Comment:
Formatting is kinda fucked on this, change it to like `jobs: [JOBS.BLACK_MAGE,],` if you maintain just a single job, for consistency sake.

Reply:
It used to not be like this :blobsweat: I fixed everything now tho